conda/plugins/subcommands/doctor/__init__.py

```diff
@@ -0,0 +1,3 @@
+# Copyright (C) 2012 Anaconda, Inc
+# SPDX-License-Identifier: BSD-3-Clause
+from . import health_checks  # noqa F401
```

Comment:
This import is not needed as it's importing the health checks module from the local namespace, the submodule `health_checks`.